recipes/open62541/all/conandata.yml

```diff
@@ -0,0 +1,18 @@
+sources:
+  "1.0.0":
```

Comment:
1.0.0 can be removed, I think.
Nobody will use an outdated version

Reply:
Last version is 1.1 anyway

Reply:
There are people who use older versions due to compatibility reasons, and it does not cause problems to keep it as an option